style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
- [ ] Please use the [Microverse template](https://github.com/microverseinc/readme-template) to document your repo, I notice that the main branch already has it, so merging this branch will cause changing the whole REAMDE content in the main branch, so Kindly suggest adding The Microverse  README to this branch as well.